root/dataframe/CMakeLists.txt

```diff
@@ -24,19 +24,22 @@ ROOTTEST_ADD_TEST(test_trainCache
 # ROOT-9628
 ROOTTEST_ADD_TEST(test_chainZombieFile
                   COPY_TO_BUILDDIR input1.root input2.root
+                  ${MIGHT_FAIL}
```

Comment:
the variable name is misleading, should really be called `WILL_FAIL_ON_WIN` or similar

Reply:
It's already user all over the place, but I'll change them all...

Reply:
thanks. it's terrible to have most RDF tests marked "MIGHT_FAIL", it's slightly less terrible if they are marked "WILL_FAIL_ON_WINDOWS" :)

Reply:
According to Axel, it's a known issue in llvm/clang/cling